src/base/global.h

```diff
@@ -34,15 +34,13 @@
 const char C_TORRENT_FILE_EXTENSION[] = ".torrent";
 
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 7, 0)
 template <typename T>
-constexpr typename std::add_const<T>::type &qAsConst(T &t) noexcept { return t; }
+constexpr typename std::add_const<T>::type &asConst(T &t) noexcept { return t; }
 
 // prevent rvalue arguments:
```

Comment:
This comment has became incorrect now. We don't really prevent rvalues now. This deleted template actually prevents only arguments declared as `const Container func(...)`.

Reply:
"prevent const rvalue arguments" is more correct, IIRC.

Reply:
Done.